src/Datastore/EntityMapper.php

```diff
@@ -429,4 +446,18 @@ private function isEncoded($value)
 
         return true;
     }
+
+    /**
+     * Get the value type from a value object.
+     *
+     * @param array $value
+     * @return string
+     */
+    private function getValueType(array $value)
+    {
+        $keys = array_keys($value);
+        return array_values(array_filter($keys, function ($key) {
+            return in_array($key, $this->valueTypes);
+        }))[0];
```

Comment:
Yep, I eliminated the static list of types in the most recent change. (GH's new review tools are really inadequate in showing the most recent changes. 😞  )